.gitattributes

```diff
@@ -0,0 +1,191 @@
+## GITATTRIBUTES FOR WEB PROJECTS
+#
+# These settings are for any web project.
+#
+# Details per file setting:
+#   text    These files should be normalized (i.e. convert CRLF to LF).
+#   binary  These files are binary and should be left untouched.
+#
+# Note that binary is a macro for -text -diff.
+######################################################################
+
+## AUTO-DETECT
+##   Handle line endings automatically for files detected as
+##   text and leave all files detected as binary untouched.
+##   This will handle all files NOT defined below.
+* text=auto
+
+## SOURCE CODE
+*.bat      text eol=crlf
+*.coffee   text
+*.css      text
+*.htm      text
+*.html     text
+*.inc      text
+*.ini      text
+*.js       text
+*.json     text
+*.jsx      text
+*.less     text
+*.od       text
+*.onlydata text
+*.php      text
+*.pl       text
+*.py       text
+*.rb       text
+*.sass     text
+*.scm      text
+*.scss     text
+*.sh       text eol=lf
```

Comment:
Many of our files have the extension `.bash`, and one has the extension `.fish`. Do those extensions need to be added here as well?

Reply:
Yes, I'll update the file extensions we have in our repo

Reply:
L14 covers anything not explicitly defined here.

Reply:
`*.sh       text` will default to LF in the git object tree, but not in working directory. So check-in will always be LF and checkout will be system dependent.